### DISCUSSION TITLE
Missing forge-std library after installation (smartcontractkit)

### DISCUSSION THREAD

**Discussion by chapman91** [Cyfrin/foundry-full-course-cu#2084] [Q&A] [answered]
I'm encountering an issue with the forge-std library in my Foundry project. I've attempted to install it using the following command: 

forge install https://github.com/smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit

I receive errors indicating that the Solidity compiler (Solc) cannot find the forge-std/Script.sol and forge-std/Test.sol files.

I examined the directory structure and the folder was not installed during the process of retrieving the smartcontractkit into my local environment set up 

Please help me to understand why this error happened. Thank you!

**Comment by chapman91** [comment, marked as answer]
I. I added the lib folder to my .gitignore file in my root directory.

II. I reinstalled forge-std library from the smartcontractkit using this command: forge install --no-commit foundry-rs/forge-std

**Reply by EngrPips** [reply]
I am glad you figured this out. Keep crushing it, buddy

**Reply by Lukelr12** [reply]
you have sprinkled me with wisdom from your mighty brain, thank you.

**Reply by EngrPips** [reply]
Looking forward to us all being successful.
